config/environments/development.rb

```diff
@@ -44,7 +44,9 @@
   # Highlight code that triggered database queries in logs.
   config.active_record.verbose_query_logs = true
 
-
+  ENV['USERNAME']="ansh-k"
+  ENV['REPOSITORY']="bookstore-api"
+  ENV['OAUTH']="INSERT OAUTH KEY"
```

Comment:
Modifications to the `ENV` hash is generally considered bad practice, as it obfuscates where the values are actually coming from (e.g. without reading this config file, if I try to run this code with `USERNAME`, `REPOSITORY` etc set in my terminal I'll be very confused as to why my passed in values aren't working). 

Furthermore, this is relying on the dev running the code to manually edit this file to insert their oauth key, and since this file is checked into Git sooner or later someone will accidentally commit that change, exposing their credentials to the internet.

Consider using something like [foreman](https://github.com/ddollar/foreman) with a gitignored `.env` file to provide development environment ENV variables instead.

Reply:
Here used `.env` with `.gitignore` this file